integration/clean/clean_ami.go

```diff
@@ -8,27 +8,19 @@ package main
 
 import (
 	"context"
-	"log"
-	"time"
-
 	"github.com/aws/aws-sdk-go-v2/aws"
 	"github.com/aws/aws-sdk-go-v2/config"
 	"github.com/aws/aws-sdk-go-v2/service/ec2"
 	"github.com/aws/aws-sdk-go-v2/service/ec2/types"
 	smithyTime "github.com/aws/smithy-go/time"
+	"log"
+	"time"
 )
 
-func main() {
-	err := cleanAMI()
-	if err != nil {
-		log.Fatalf("errors cleaning %v", err)
-	}
-}
+var expirationDateAMI = time.Now().UTC().Add(keepDurationAMI)
```

Comment:
I'm not a fan of defining the expiration date at the top like this, depending on `time.Now()`. Can you add a `time.Time` parameter to the `cleanAMI` function, and then do this computation inside the function?